integration/sawtooth_integration/tests/test_intkey_smoke.py

```diff
@@ -89,7 +89,10 @@ def post_and_verify(self, batch, how_many_updates):
         batch = IntkeyMessageFactory().create_batch(batch)
         LOGGER.info('Posting batch')
         _post_batch(batch)
-        time.sleep(1)
+
+        while (how_many_updates + 1) > _get_block_num():
```

Comment:
It should timeout if it stops working.

We can't check for failures until the rest API can report invalid batches.